pkg/sdk/warehouses.go

```diff
@@ -483,6 +504,9 @@ func (row warehouseDBRow) convert() *Warehouse {
 	if row.OwnerRoleType.Valid {
 		wh.OwnerRoleType = row.OwnerRoleType.String
 	}
+	if row.ResourceMonitor != "null" {
```

Comment:
`|| row.ResourceMonitor != ""` ?

Reply:
Had it before but it is not necessary -> will result in the same struct being created